client/components/AutocompleteInput/AutocompleteInput.js

```diff
@@ -53,16 +84,34 @@ export default class AutocompleteInput extends PureComponent {
     onSearchSubmit(value || this.state.value)
   }
 
-  handleInputChange = ({ target }) => {
-    this.setState({ value: target.value })
-    const trimmedValue = target.value.trim()
-    const { name } = parsePackageString(trimmedValue)
+  comparePackages = (item) => {
+    const { value } = this.state;
 
-    if (trimmedValue.length > 1) {
-      this.getSuggestions(name)
+    if (value === item.package.name) return
+
+    if (isComparingPackages(value)) {
+      const commaSeperatedValue = value.split(',');
+      commaSeperatedValue.pop();
+      this.setState({ value: `${commaSeperatedValue.join(',')},${item.package.name},` });
     }
   }
 
+  handleInputChange = ({ target }) => {
+    const { value } = target;
+    this.setState({ value: target.value  });
```

Comment:
value is already destructured from the target. can we do ` this.setState({ value  })`?

Reply:
Yep my bad. I didn't notice it.